hedera-node/src/main/java/com/hedera/services/store/models/Token.java

```diff
@@ -180,7 +182,8 @@ public void wipe(OwnershipTracker ownershipTracker, TokenRelationship accountRel
 
 		validateTrue(serialNumbers.size() > 0, FAIL_INVALID, () ->
 				"Cannot wipe " + serialNumbers.size() + " number of Unique Tokens.");
-		for (UniqueToken token : loadedUniqueTokens) {
+		for (Long serialNum : serialNumbers) {
+			var token = loadedUniqueTokens.get(serialNum);
```

Comment:
nit: lets use `uniqueToken` instead of `token`. Token refers to the class not the nft and it might be misleading